### PR TITLE
Include all active sessions in the web API

### PR DESCRIPTION
**Pull request by zmb3**
Depends on gravitational/webapps#1315

Updates #17871

### CODE REVIEW

File: api/types/session_tracker.go [hunk 1]
```diff
@@ -99,6 +99,15 @@ type SessionTracker interface {
 	// GetKubeCluster returns the name of the kubernetes cluster the session is running in.
 	GetKubeCluster() string
 
+	// GetDesktopName returns the name of the Windows desktop the session is running in.
+	GetDesktopName() string
```
Comment (kimlisa) [inline api/types/session_tracker.go:103]: weird, i don't see this being a duplicate as `github actions/analyze` says it is...

File: lib/web/apiserver_test.go [hunk 2]
```diff
@@ -1406,46 +1406,80 @@ func TestWebAgentForward(t *testing.T) {
...
 	s := newWebSuite(t)
-	sid := session.NewID()
 	pack := s.authPack(t, "foo")
 
-	ws, err := s.makeTerminal(t, pack, withSessionID(sid))
-	require.NoError(t, err)
-	t.Cleanup(func() { require.NoError(t, ws.Close()) })
+	start := time.Now()
+	kinds := []types.SessionKind{
+		types.SSHSessionKind,
```
Comment (kimlisa) [inline lib/web/apiserver_test.go:1413]: add `KubernetesSessionKind` while at it?

**Review by kimlisa — APPROVED** [review]

**Review by rosstimothy — APPROVED** [review]

File: lib/web/apiserver.go [hunk 3]
```diff
@@ -2209,6 +2210,9 @@ func trackerToLegacySession(tracker types.SessionTracker, clusterName string) se
 		ServerAddr:            tracker.GetAddress(),
 		ClusterName:           clusterName,
 		KubernetesClusterName: tracker.GetKubeCluster(),
+		DesktopName:           tracker.GetDesktopName(),
+		AppName:               tracker.GetAppName(),
+		DatabaseName:          tracker.GetDesktopName(),
```
Comment (greedy52) [inline lib/web/apiserver.go:2215]: should this be `GetDatabaseName`?

Comment (zmb3) [reply lib/web/apiserver.go:2215]: 🤦‍♂️

**Review by greedy52 — APPROVED** [review]

**Pull request merged**